python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -1008,6 +1008,8 @@ class OpenAIChatCompletionClient(BaseOpenAIChatCompletionClient, Component[OpenA
 
         client = ChatCompletionClient.load_component(config)
 
+    Note: When usage information is requested (see `documentation <https://platform.openai.com/docs/api-reference/chat/streaming#chat/streaming-choices>`_.) with the `create_stream` method, `max_consecutive_empty_chunk_tolerance` should be increased to permit the trailing empty chunk carrying the usage information.  E.g. `completion_client.create_stream(... , max_consecutive_empty_chunk_tolerance=2, extra_create_args={"stream_options": {"include_usage": True}})`.
```

Comment:
Use tip directive: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives

Reply:
Also, apologize for the confusion in my comment in a different issue. I think this should go to the API doc of `create_stream` method in the `BaseOpenAIChatCompletionClient`.